infra/base-images/base-runner/bad_build_check

```diff
@@ -44,6 +44,12 @@ UBSAN_CALLS_THRESHOLD_FOR_UBSAN_BUILD=170
 # a big concern either way as the overhead for them would not be significant.
 UBSAN_CALLS_THRESHOLD_FOR_NON_UBSAN_BUILD=200
 
+# ASan builds on i386 generally have about 250 UBSan runtime calls.
```

Comment:
Why do we use 280 below instead of 250 ? @Dor1s does this look ok ?

Reply:
I don't know, this is what I observed in a couple different binaries. I don't think we should spend too much time on this as it is a rule of thumb not a hard rule.

Reply:
SG